### PR TITLE
Updates for revscoring 2.3.0

### PR DESCRIPTION
**Pull request by halfak**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -87,33 +89,33 @@ mysql_args = -h $(host) -u $(user)
...
-	  bzcat datasets/enwiki.draft_quality.201512.tsv.bz2 | tail -n+2; \
-	  bzcat datasets/enwiki.draft_quality.201601.tsv.bz2 | tail -n+2; \
-	  bzcat datasets/enwiki.draft_quality.201602.tsv.bz2 | tail -n+2; \
-	  bzcat datasets/enwiki.draft_quality.201603.tsv.bz2 | tail -n+2; \
-	  bzcat datasets/enwiki.draft_quality.201604.tsv.bz2 | tail -n+2; \
-	  bzcat datasets/enwiki.draft_quality.201605.tsv.bz2 | tail -n+2; \
-	  bzcat datasets/enwiki.draft_quality.201606.tsv.bz2 | tail -n+2; \
-	  bzcat datasets/enwiki.draft_quality.201607.tsv.bz2 | tail -n+2 \
-	) | bzip2 -c > $@
+# datasets/enwiki.draft_quality.201508-201608.tsv.bz2: \
```
Comment (adamwight) [inline Makefile:92]: Can you comment about this?  There should be a reason given in the source code, or the block should be removed.

**Comment by adamwight** [comment]
Merging despite comment request.

File: Makefile (hunk 1 above)
Comment (halfak) [reply Makefile:92]: Oh!  Yes.  This is confusing.  The process that extracts the draft quality data is super-long running.  It's safer to keep this block commented out than to accidentally overwrite an expensive file.  Is there are better way to keep the Makefile code so we know what command to run but to not let it automatically execute during a make call? 

If not, I'll submit a followup PR for putting a comment above this.

Comment (adamwight) [reply Makefile:92]: Thanks for the note!  There are a few ways to deal with this, but I don't fully understand what causes the expensive file to be overwritten, nor which is the expensive file here.

Depending on the root issue here, one tool we might use is "order only" dependencies, see https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html — essentially, we just add a pipe "|" before the list of dependencies and then the output file won't be rebuilt unless it's missing.

From reading the Makefile I think this is the issue you're referring to, let me know…

**Pull request merged**